tests/spec/fs.chmod.spec.js

```diff
@@ -39,6 +39,20 @@ describe('fs.chmod, fs.fchmod', function() {
     });
   });
 
+  it('should be a valid mode value to set permissions', function(done){
+    var fs = util.fs();
+
+    fs.mkdir('/file', function(err) {
+      if(err) throw err;
+      
+      fs.chmod('file', 1000, function(err) {
```

Comment:
This is valid in node.js (i.e., you can use a mode of `1000`).  Same for `"1000"`, which node will turn into an integer.  If you want to make this test fail, you could use `{}`:

```js
fs.chmod('file', {}, function(err) {
  expect(err).to.exist;
  expect(err.code).to.equal('EINVAL');
  done();
});
```

node will give you a `TypeError [ERR_INVALID_ARG_TYPE]: The "mode" argument must be of type number. Received type object` which we should probably match.